ci/infra/testrunner/tests/test_dockercaps.py

```diff
@@ -0,0 +1,47 @@
+import logging
+import pytest
+import re
+import time
+
+logger = logging.getLogger("testrunner")
+
+
+class NoRootUserError(RuntimeError):
+    def __init__(self, container, output):
+        if 'su' in output:
+            msg = "Container {}: {} -- Regression (see bsc#1142552).".format(container, output)
+        else:
+            msg = "No root user in container {}: {}".format(container, output)
+        super().__init__(msg)
+
+class CannotAddUser(RuntimeError):
+    def __init__(self, container, output):
+        msg = "Cannot add a new user in the container {}: {}".format(container, output)
+        super().__init__(msg)
+
+
+def test_dockercaps(deployment, kubectl):
+
+    logger.info("Deploy testcases")
+    kubectl.run_kubectl(
+        "apply -f https://pastebin.com/raw/iNA7Kh9L")
+    kubectl.run_kubectl("wait --for=condition=ready pods --all --timeout=3m")
+
+    logger.info("Test: Run 'su root -c id' on the containers")
+    pods = ["sle12sp4", "leap", "sle15", "sle15sp1"]
+    for container in pods:
+        output = kubectl.run_kubectl(
+            "exec -it {} -- {}".format(container, 'su root -c id'))
```

Comment:
Not sure about using a constant string as a substitution in another constant string. I would put `su root -c id` as part of the `exec ....`